crates/ibc-types-identifier/Cargo.toml

```diff
@@ -23,6 +23,7 @@ std = [
     "serde/std",
     "displaydoc/std",
 ]
+with_serde = ["serde", "serde_derive"]
```

Comment:
weirdly enough, aliasing the dep tree: `serde = ["dep:serde", "dep:serde_derive"]` was not sufficient, the compiler was not able to resolve the `serde::Serialize` dependency on `IdentifierMsg`, ditto elsewhere.